pkg/service/generateConfig/generateConfig.go

```diff
@@ -57,49 +58,22 @@ test:
   passThroughPorts: []
   withCoverage: false
   coverageReportPath: ""
-  #
-  # Example on using globalNoise
-  # globalNoise: 
-  #    global:
-  #      body: {
-  #         # to ignore some values for a field, 
-  #         # pass regex patterns to the corresponding array value
-  #         "url": ["https?://\S+", "http://\S+"],
-  #      }
-  #      header: {
-  #         # to ignore the entire field, pass an empty array
-  #         "Date: [],
-  #       }
-  #     # to ignore fields or the corresponding values for a specific test-set,
-  #     # pass the test-set-name as a key to the "test-sets" object and
-  #     # populate the corresponding "body" and "header" objects 
-  #     test-sets:
-  #       test-set-1:
-  #         body: {
-  #           # ignore all the values for the "url" field
-  #           "url": []
-  #         }
-  #         header: { 
-  #           # we can also pass the exact value to ignore for a field
-  #           "User-Agent": ["PostmanRuntime/7.34.0"]
-  #         }
-`
+  `
 
 func (g *generatorConfig) GenerateConfig(filePath string) {
 	var node yaml.Node
-
 	data := []byte(config)
-
 	if err := yaml.Unmarshal(data, &node); err != nil {
 		g.logger.Fatal("Unmarshalling failed %s", zap.Error(err))
 	}
-
 	results, err := yaml.Marshal(node.Content[0])
 	if err != nil {
 		g.logger.Fatal("Failed to marshal the config", zap.Error(err))
 	}
 
-	err = os.WriteFile(filePath, results, os.ModePerm)
+  finalOutput := append(results, []byte(utils.ConfigGuide)...)
```

Comment:
Is this tested that we are getting same config file as before..?

Reply:
<img width="594" alt="image" src="https://github.com/keploy/keploy/assets/37413698/d0206c4c-fdba-46af-8976-a022f6c2b932">
yes, attaching screenshot for reference